src/types.ts

```diff
@@ -7,10 +7,8 @@ export interface SendbirdChatAICallbacks {
   onWidgetSettingFailure?: (error: Error) => void;
 }
 
-export interface FunctionCallRequestInfo {
-  headers: {
-    'Api-Token': string;
-  };
+export interface FunctionCallRequest {
+  headers: object;
```

Comment:
is there any particular reason why type changed to object?

Reply:
The header varies depending on the function call set by the customer, just like other payloads.